tlsconfig/config.go

```diff
@@ -83,6 +84,22 @@ func certPool(caFile string) (*x509.CertPool, error) {
 	return certPool, nil
 }
 
+// minVersion returns the minimum TLS version matching 'version', the MinVersion string.
+func minVersion(version string) (uint16, error) {
+	allVersions := map[string]uint16{
+		"VersionSSL30": tls.VersionSSL30,
```

Comment:
I'm not sure that the string translation should happen in the `go-connections` library.  Agree that there doesn't seem to be any standard.  Probably `go-connections` should just take a `uint16` like `tls.Config`, and enforce that that number is at least >= `tls.VersionTLS10`?

The user-friendly translation can happen at the user-facing layer (like a CLI or config file parser).